recipes/zstd/all/conanfile.py

```diff
@@ -51,41 +47,39 @@ def configure(self):
         del self.settings.compiler.cppstd
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
-
-    def _configure_cmake(self):
-        if self._cmake:
-            return self._cmake
-        self._cmake = CMake(self)
-        self._cmake.definitions["ZSTD_BUILD_PROGRAMS"] = False
-        self._cmake.definitions["ZSTD_BUILD_STATIC"] = not self.options.shared
-        self._cmake.definitions["ZSTD_BUILD_SHARED"] = self.options.shared
-        self._cmake.definitions["ZSTD_MULTITHREAD_SUPPORT"] = self.options.threading
-        if tools.Version(self.version) < "1.4.3":
-            # Generate a relocatable shared lib on Macos
-            self._cmake.definitions["CMAKE_POLICY_DEFAULT_CMP0042"] = "NEW"
```

Comment:
why this one is removed?

Reply:
Good catch, have placed it back after reviewing the older versions.